xilem_web/src/attribute.rs

```diff
@@ -61,6 +61,15 @@ impl Attributes {
 }
 
 fn set_attribute(element: &web_sys::Element, name: &str, value: &str) {
+    debug_assert_ne!(
+        name, "class",
+        "Using `class` as attribute is not supported, use the `el.class()` modifier instead"
+    );
+    debug_assert_ne!(
+        name, "style",
+        "Using `style` as attribute is not supported, use the `el.style()` modifier instead"
+    );
```

Comment:
These are some of the details and possibilities that I love about rust. A very good usecase in my opinion :+1: